### PR TITLE
+angstrom.0.12.0

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/angstrom-unix/angstrom-unix.0.12.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
+  "angstrom"
```
Comment (kit-ty-kate) [inline packages/angstrom-unix/angstrom-unix.0.12.0/opam]:
```suggestion
  "angstrom" {= version}
```
Are you sure that doesn't need a version constraint?

File: packages/angstrom-lwt-unix/angstrom-lwt-unix.0.12.0/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
+  "angstrom"
```
Comment (kit-ty-kate) [inline packages/angstrom-lwt-unix/angstrom-lwt-unix.0.12.0/opam]:
```suggestion
  "angstrom" {= version}
```
Are you sure that doesn't need a version constraint?

File: packages/angstrom-async/angstrom-async.0.12.0/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "dune" {>= "1.0"}
+  "angstrom" {>= "0.7.0"}
```
Comment (kit-ty-kate) [inline packages/angstrom-async/angstrom-async.0.12.0/opam]:
```suggestion
  "angstrom" {= version}
```
Are you sure it works with such an old version? I'd more recommend to stick with the same version for the same "set of packages".

**Comment by kit-ty-kate** [comment]
There is a revdep failure which seems related in `git.2.0.0` and `git.2.1.0`:
```
+24008us git.encoder [DEBUG] Ensure than we wrote exactly [shift = 206] byte(s).
# [invalid] Bigstringaf.substring invalid range: { buffer_len: 206, off: 200, len: 206 }
# Raised at file "lib/bigstringaf.ml", line 45, characters 2-34
# Called from file "lib/bigstringaf.ml", line 98, characters 7-52
# Called from file "lib/input.ml" (inlined), line 68, characters 2-22
# Called from file "lib/angstrom.ml", line 367, characters 39-83
# Called from file "lib/parser.ml", line 38, characters 18-59
# Called from file "lib/angstrom.ml" (inlined), line 135, characters 2-33
# Called from file "src/git/helper.ml", line 231, characters 4-60
# Called from file "src/git/value.ml" (inlined), line 482, characters 36-65
# Called from file "test/common/test_store.ml", line 319, characters 14-64
# Called from file "test/common/test_store.ml" (inlined), line 55, characters 30-39
# Called from file "test/common/test_store.ml", line 346, characters 4-12
# Called from file "src/alcotest.ml", line 285, characters 8-14
```
cc @dinosaure

**Comment by kit-ty-kate** [comment]
I think there is a similar issue in `emile.0.2` to `emile.0.5`. cc @dinosaure

**Comment by seliopou** [comment]
It does look like there is a problem with this release related to inhabitedtype/angstrom#181. I think I know what it is so I'll close this for now and submit another version with the fix shortly.

**Pull request closed without merging**